Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -299,10 +308,10 @@ public struct RNN<Cell: RNNCell>: Layer {
           (Array<Cell.TimeStepOutput>.TangentVector)
               -> (TangentVector, Array<Cell.TimeStepInput>.TangentVector)) {
         let timeStepCount = inputs.count
-        var currentHiddenState = cell.zeroState
-        var timeStepOutputs: [Cell.TimeStepOutput] = []
+        var currentHiddenState = cell.zeroState(for: inputs[0])
+        var timeStepOutputs = [Cell.TimeStepOutput]()
         timeStepOutputs.reserveCapacity(timeStepCount)
-        var backpropagators: [Cell.Backpropagator] = []
+        var backpropagators = [Cell.Backpropagator]()
```

Comment:
We actually prefer literal initialization for arrays in this code base. Could you revert it?

Reply:
Sure, I wasn't aware. :)